src/Sobol.jl

```diff
@@ -122,11 +122,19 @@ struct ScaledSobolSeq{N} <: AbstractSobolSeq{N}
     s::SobolSeq{N}
     lb::Vector{Float64}
     ub::Vector{Float64}
-    ScaledSobolSeq(lb::Vector{Float64}, ub::Vector{Float64}) =
+    function ScaledSobolSeq(N::Integer, lb::Vector{<:Real}, ub::Vector{<:Real})
```

Comment:
The right thing to do is to use `ScaledSobolSeq{N}(...) where {N}` for the inner constructor here.  (This changed in Julia 0.7, which is why this code stopped working.)